src/modules/features/selectors.ts

```diff
@@ -54,6 +54,14 @@ export const getIsPublishSmartWearablesEnabled = (state: RootState) => {
   }
 }
 
+export const getIsNewNavbarDropdownEnabled = (state: RootState) => {
+  try {
+    return getIsFeatureEnabled(state, ApplicationName.BUILDER, FeatureName.NEW_NAVBAR_DROPDOWN)
+  } catch (e) {
+    return false
+  }
+}
+
```

Comment:
Could you add a test for this? You'll need to only add a new row in the `ffSelectors` array in the `features/selectors.spec.ts` file.